themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties

```diff
@@ -0,0 +1,27 @@
+emailTestSubject=[KEYCLOAK] - SMTP testbericht
+emailTestBody=Dit is een testbericht
+emailTestBodyHtml=<p>Dit is een testbericht</p>
+identityProviderLinkSubject=Koppel {0}
+identityProviderLinkBody=Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen\n\n{3}\n\nDeze link zal over {4} minuten vervallen.\n\nIndien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar {1} inloggen via {0}.
+identityProviderLinkBodyHtml=<p>Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen</p><p><a href="{3}">Link om accounts te koppelen</a></p><p>Deze link zal over {4} minuten vervallen.</p><p>Indien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar {1} inloggen via {0}.</p>
+passwordResetSubject=Wijzig wachtwoord
+passwordResetBody=Iemand heeft zojuist gevraagd om de aanmeldgegevens van {2} account te wijzigen. Indien u dit was, klik dan op de onderstaande koppeling om ze te wijzigen.\n\n{0}\n\nDe link en de code zullen binnen {1} minuten vervallen.\n\nIndien u uw aanmeldgegevens niet wilt wijzigen, negeert u dan dit bericht en er zal niets veranderd worden.
+passwordResetBodyHtml=<p>Iemand heeft zojuist gevraagd om de aanmeldgegevens van {2} account te wijzigen. Indien u dit was, klik dan op de onderstaande koppeling om ze te wijzigen.</p><p><a href="{0}">Wijzig aanmeldgegevens</a></p><p>De link en de code zullen binnen {1} minuten vervallen.</p><p>Indien u uw aanmeldgegevens niet wilt wijzigen, negeert u dan dit bericht en er zal niets veranderd worden.</p>
+executeActionsSubject=Wijzig uw account
+executeActionsBody=Uw beheerder heeft u zojuist verzocht uw {2} account te wijzigen. Klik op de onderstaande koppeling om dit proces te starten. \n\n{0}\n\nDeze link zal over {1} minuten vervallen. \n\nIndien u over dit verzoek niet op de hoogte was, negeert u dan dit bericht en er zal niets worden veranderd.
+executeActionsBodyHtml=<p>Uw beheerder heeft u zojuist verzocht uw {2} account te wijzigen. Klik op de onderstaande koppeling om dit proces te starten.</p><p><a href="{0}">Link naar account wijziging</a></p><p>Deze link zal over {1} minuten vervallen.</p><p>Indien u over dit verzoek niet op de hoogte was, negeert u dan dit bericht en er zal niets worden veranderd.</p>
+eventLoginErrorSubject=Inlogfout
+eventLoginErrorBody=Er is een mislukte inlogpoging gedetecteerd op uw account om {0} vanuit {1}. Als u dit niet was, neemt u dan contact op met de beheerder.
+eventLoginErrorBodyHtml=<p>Er is een mislukte inlogpoging gedetecteerd op uw account om {0} vanuit {1}. Als u dit niet was, neemt u dan contact op met de beheerder.</p>
+eventRemoveTotpSubject=TOTP verwijderd
```

Comment:
I agree, but again I tried to keep close to the other translations (both English, but also all other languages). I suggest we don't change it here, but have a change request (JIRA issue) to modify all translations.